applications/vessel-history/src/features/search/search.slice.ts

```diff
@@ -24,8 +24,6 @@ export type SearchSlice = {
   queries: CachedQuerySearch
 }
 
-export type SearchType = 'basic' | 'advanced'
```

Comment:
👍  should we also remove it from the workspace params https://github.com/GlobalFishingWatch/frontend/blob/3807ba41c972052650d0ad524c96407bc4da85ad/applications/vessel-history/src/types/index.ts#L6 ?